src/main/java/seedu/address/logic/commands/HelpCommand.java

```diff
@@ -10,12 +10,26 @@
     public static final String COMMAND_WORD = "help";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
-            + "Example: " + COMMAND_WORD;
+            + "Example: " + COMMAND_WORD + " [COMMAND]";
 
     public static final String SHOWING_HELP_MESSAGE = "Opened help window.";
 
+    private final CommandEnum commandToPrint;
+
+    public HelpCommand() {
+        this.commandToPrint = null;
+    }
+
+    public HelpCommand(CommandEnum commandToPrint) {
+        this.commandToPrint = commandToPrint;
+    }
+
     @Override
     public CommandResult execute(Model model) {
-        return new CommandResult(SHOWING_HELP_MESSAGE, true, false);
+        if (commandToPrint == null) {
```

Comment:
Usage of null here but I believe there is not much choice here. I believe you have taken care of trimming the preamble too - eg. help "    " was keyed. Nice job 😎